include/mbedtls/config.h

```diff
@@ -2211,18 +2211,32 @@
  */
 #define MBEDTLS_PKCS5_C
 
+/**
+ * \def MBEDTLS_PKCS11_CLIENT_C
+ *
+ * Enable support for keys stored in an external token, using the
+ * cryptoki (PKCS#11) interface.
+ *
+ * Module:  library/pkcs11_client.c
+ *
+ * This module requires a PKCS#11 library.
+ *
+ */
+#define MBEDTLS_PKCS11_CLIENT_C
```

Comment:
Note: this should be disabled by default as it adds a new dependency on an external library. I suggest doing this change just before merging in the main branch, but please remember to do it.

Reply:
Also the CI-s doesn't seem to be prepared for this and I guess this is why all of them are failing.

Reply:
Yes and yes. It's enabled by default in the branch because we're almost always building with the feature enabled, but it will be disabled by default in the product because of the external dependency. A subsequent PR will add what it takes to test on CI, at the moment we're only testing locally.